indy_node/utils/node_control_tool.py

```diff
@@ -81,6 +86,48 @@ def __init__(
         # Listen for incoming connections
         self.server.listen(1)
 
+    def _ext_init(self):
+        self.ext_upgrade = self.config.UPGRADE_ENTRY not in PACKAGES_TO_HOLD
+        self._update_package_cache()
+        self.ext_ver, ext_deps = self._ext_info()
+        self.deps = ext_deps + self.deps
```

Comment:
Please add a test that dependencies are resolved correctly if indy-node is one of the dependencies for top-level package.
See `testNodeControlResolvesDependencies` as example.